comp/otelcol/ddprofilingextension/impl/extension.go

```diff
@@ -87,13 +106,41 @@ func (e *ddExtension) startForOCB() error {
 		profiler.WithAPIKey(string(e.cfg.API.Key)),
 	)
 
-	// todo(mackjmr): add datadogexporter hostmetadata provider to retrieve hostname, and
-	// pass it to profiler via profiler.WithHostname(). This requires a refactor in contrib,
-	// as the logic lives within an internal package.
 	if string(e.cfg.API.Site) != "" {
 		profilerOptions = append(profilerOptions, profiler.WithSite(string(e.cfg.API.Site)))
 	}
 
+	source, err := e.sourceProvider.Source(context.Background())
+	if err != nil {
+		return err
+	}
+
+	var tags strings.Builder
+	// hardcode agent version to 7.64.0. The backend requires this tag, but this path does not include agent,
+	// so hardcode value.
+	tags.WriteString("agent_version:7.64.0")
```

Comment:
Is this expected to be the version moving forward?

Reply:
I've tested this and the version ends up appearing in the profiles, so I don't think this is a good approach, it could lead to confusion if you are using agent ex v 7.65 and the profiles show 7.64. I'll try to find a better way.

Reply:
I've updated this in https://github.com/DataDog/datadog-agent/pull/35004/commits/bccba7da215087dcb7831e28745c02aa6dc6d400 to use the version of `github.com/DataDog/datadog-agent/comp/trace/agent/def ` instead, checking with profiling team if this is OK